graphql_compiler/schema_generation/orientdb/get_orientdb_schema_graph.py

```diff
@@ -0,0 +1,462 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from enum import Enum
+from itertools import chain
+
+from funcy.py3 import lsplit
+from graphql.type import GraphQLList, GraphQLObjectType
+import six
+
+from ..exceptions import IllegalSchemaStateError
+from ..schema_graph import (
+    EdgeType, NonGraphElement, SchemaGraph, VertexType, get_subclass_sets_from_inheritance_sets
+)
+from .schema_properties import (
+    COLLECTION_PROPERTY_TYPES, EDGE_DESTINATION_PROPERTY_NAME, EDGE_END_NAMES,
+    EDGE_SOURCE_PROPERTY_NAME, ORIENTDB_BASE_EDGE_CLASS_NAME, ORIENTDB_BASE_VERTEX_CLASS_NAME,
+    PROPERTY_TYPE_LINK_ID, PropertyDescriptor, get_graphql_scalar_type_or_raise,
+    parse_default_property_value
+)
+from .utils import toposort_classes
+
+
+class Kind(Enum):
```

Comment:
I don't love this Kind Enum, but its simple and doesn't affect code quality that much. Another alternative would be to use the EdgeType, VertexType and NonGraphElement classes directly as the kind.